config/manager/basicsecret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: secret-basic-auth
+  namespace: numaplane-system
+type: kubernetes.io/basic-auth
+stringData:
+  username: "admin" # required field for kubernetes.io/basic-auth
+  password: "t0p-Secret" # required field for kubernetes.io/basic-auth
```

Comment:
this Secret is maybe okay to go in the `samples` directory. Let's not put it in here, since this is what will be deployed by default.